apps/dialtone-documentation/docs/utilities/typography/font-variant-numeric.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Font variant numeric
+description: Utilities to change an element's font variant numeric.
+---
+
+The `font-variant-numeric` CSS property controls the usage of alternate glyphs for numbers, fractions, and ordinal markers.
+
+## Normal (default)
+
+Use `d-fvn-normal` to deactivate any alternate glyphs.
+
+<code-well-header class="d-d-flex d-jc-center d-fd-column d-p24 d-bgc-purple-100 d-w100p d-hmn102" custom>
+  <p class="d-fs-300 d-fc-purple-400 d-fvn-normal">1234567890</p>
+</code-well-header>
+
+```html
+<p class="d-fs-300 d-fvn-normal">...</p>
+```
+
+## Ordinal
+
+The class `d-fvn-ordinal` forces the use of special glyphs for ordinal markers, like 1st, 2nd, 3rd.
+
+<code-well-header class="d-d-flex d-jc-center d-fd-column d-p24 d-bgc-purple-100 d-w100p d-hmn102" custom>
+  <p class="d-fs-300 d-fc-purple-400 d-fvn-ordinal">1st, 2nd, 3rd, 4th</p>
+</code-well-header>
+
+```html
+<p class="d-fs-300 d-fvn-ordinal">...</p>
+```
+
+## Proportional-nums
+
+Use `d-fvn-proportional` to set different sizes for each number.
```

Comment:
Is this the default in a table?

Reply:
No, the default is `normal`. For some fonts, you might see a change between the two.